### DISCUSSION TITLE
README.MD relative file links are broken in the auto-preview?

### DISCUSSION THREAD

**Discussion by nbouvrette** [community/community#70735] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I just noticed a broken link on one of my repository: https://github.com/Avansai/properties-file

If you check the link in this sentence:

> Please refer to the [upgrade guide](https://github.com/Avansai/V2-TO-V3-UPGRADE-GUIDE.md) before upgrading.

When looking at the readme, from the autopreview from https://github.com/Avansai/properties-file, the link points to https://github.com/Avansai/V2-TO-V3-UPGRADE-GUIDE.md rather than https://github.com/Avansai/properties-file/V2-TO-V3-UPGRADE-GUIDE.md

If I do the same test but opening the README.MD file here: https://github.com/Avansai/properties-file/blob/main/README.md

The link points to a valid location: https://github.com/Avansai/properties-file/blob/main/V2-TO-V3-UPGRADE-GUIDE.md

The actual source for the link is `Please refer to the [upgrade guide](V2-TO-V3-UPGRADE-GUIDE.md) before upgrading.`

I also tried to use `./V2-TO-V3-UPGRADE-GUIDE.md` and the same bug occurs. I also tried `/V2-TO-V3-UPGRADE-GUIDE.md` as recommended [here](https://github.com/orgs/community/discussions/70664#discussioncomment-7343192) but the end result is even worse and generates this link both on the auto-preview and file directly: https://github.com/V2-TO-V3-UPGRADE-GUIDE.md

I am pretty sure this link was working and this looks like a new Github bug?

**Comment by acusti** [comment, marked as answer]
following up on https://github.com/orgs/community/discussions/70664#discussioncomment-7348156:

this is a duplicate of https://github.com/orgs/community/discussions/67750. the bug: relative links are broken for the repo’s root readme file as rendered at the repo homepage if they are contained within a non-paragraph markdown block.

in this case, the workarounds are:

1. remove the linked text from any non-paragraph markdown block, e.g. remove the `> ` from in front of the text containing the broken link in this case
2. update the link to have a leading slash and be built as from `github.com/*`, so the link in this case would be `/Avansai/properties-file/blob/main/V2-TO-V3-UPGRADE-GUIDE.md`. this works in my testing (both at the root of the repo and in the file browser view) but might break if/when GitHub fixes this bug.
